app/views/refinery/wymiframe.html.erb

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
```

Comment:
So were we already using Refinery's WYMeditor and this is applying a fix so that our asset pipeline is good to go on staging, or is this a totally new addition to fix aforementioned asset problem?

Reply:
Yeah we're already using WYMeditor so this just patches the asset pipeline to get this working on staging again.